### PR TITLE
 🐛 Bug Report: Wrong design for error with project name #4560

### PR DESCRIPTION
**Pull request by gitstart**
### DEMO VIDEO
https://www.loom.com/share/57be56fd96c54edd87e0330ac1fe91af.

### CODE REVIEW

**Comment by gitstart** [comment]
@GreenMachine01 This PR is ready for review. Thanks. cc @EugeneTseitlin

File: packages/amplication-client/src/Project/NewProject.tsx [hunk 1]
```diff
@@ -56,31 +58,30 @@ type Props = {
...
   const handleSubmit = useCallback(
-    (data: CreateProjectType) => {
+    (data: CreateProjectType, { setErrors }) => {
       createProject({
         variables: {
           data,
         },
-      }).catch(console.error);
+      }).catch((err) => {
+        setErrors({ name: ERROR });
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/NewProject.tsx:79]: Please keep the original error text. The current implementation will display the same error message for all types of errors that can be returned from the server which is not correct 
```suggestion
        setErrors({ name: formatError(err); });
```

File: packages/amplication-client/src/Project/NewProject.tsx [hunk 2]
```diff
@@ -56,31 +58,30 @@ type Props = {
...
+        console.error(err);
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Project/NewProject.tsx:80]: we can remove this now

**Review by yuval-hazaz — DISMISSED** [review]
Thanks for the fix! I like the way you push the error to the form. We should use this as a common practice (@overbit @abrl91 please have a look) 


Please see one comment to be fixed

**Comment by gitstart** [comment]
@yuval-hazaz  resolved the comments.

**Review by EugeneTseitlin — CHANGES_REQUESTED** [review]
Please, make sure that changes are applied only to New Project component. Merge recent changes from our repo and resolve a conflict.
The change in New Project component is fine, I've checked it and it works as expected.

**Comment by gitstart** [comment]
@EugeneTseitlin  resolved conflicts.

**Comment by EugeneTseitlin** [comment]
> @EugeneTseitlin conflicts resolved.

There should not be any change in  ISSUE_TEMPLATE, workflows folders.
Changes should be only in NewProject.tsx file.

**Comment by gitstart** [comment]
> > @EugeneTseitlin conflicts resolved.
> 
> There should not be any change in ISSUE_TEMPLATE, workflows folders. Changes should be only in NewProject.tsx file.

Hey @EugeneTseitlin reverted changes in workflow file. I think it was added somehow when we try to resolve changes between master and next branch

**Review by EugeneTseitlin — APPROVED** [review]

**Pull request merged**